### ISSUE TITLE
Enum schema with "YES" / "NO" literals may give true / false after second round of yaml parsing as not escaped.

### ISSUE THREAD

**Issue by soberich** [Jyskebank/enunciate-openapi#18]
Parent issue https://github.com/smallrye/smallrye-open-api/issues/168
Yaml reserved keywords should be put in quotes
The module now generates
```yaml
components:
  schemas:
    "Insurance":
      title: "InsuranceStatus"
      type:  string
      enum:
        - YES
        - NO
        - NEVER
```
Which have high chance of giving being understood by `yaml` format as the following (or if, say, read into memory again). https://yaml.org/type/bool.html

```yaml
components:
  schemas:
    Insurance:
      title: InsuranceStatus
      enum:
        - true
        - false
        - NEVER
      type:  string
```
THOUGH in examples are not affected and preserve "YES" / "NO" values for schemas' properties.

Format specific reserved words should be quoted/escaped.

**Comment by jskovjyskebankdk** [comment]
Fixed.
Will also fix same problem for property names and required fields.

**Comment by soberich** [comment]
I found out this seems to could've been resolved simply by `%YAML 1.2` directive at the beginning of the file.
